charts/substra-backend/templates/statefulset-builder.yaml

```diff
@@ -200,6 +200,8 @@ spec:
             value: {{ toYaml .Values.builder.kanikoStartup.maxAttempts | quote }}
           - name: BUILDER_KANIKO_STARTUP_PENDING_STATE_WAIT_SECONDS
             value: {{ toYaml .Values.builder.kanikoStartup.checkDelay | quote }}
+          - name: IMAGE_SAVING_TIMEOUT_SECONDS
+            value: {{ .Values.builder.saveImageTimeoutSeconds | quote }}
```

Comment:
Is it needed for the `builder` itself?

Reply:
In my testing, in some cases the  worker was not sufficient, will retest

Reply:
Also confirmed on my end, LGTM